packages/styled-components/components/byline/byline.stories.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+import { PureByline as Byline } from '.';
```

Comment:
oh neat I didn't realize you can just do that with an index file.

Reply:
Me neither. @ostowe showed me.